scripts/copyright/UpdateCopyrightHeader.py

```diff
@@ -60,6 +60,11 @@ def run(args):
     output = p.stdout.read()
     return output
 
+def runcode(args):
+    p = Popen(args, stderr=PIPE)
+    code = p.returncode
```

Comment:
I believe you'll want to add this after creating the `Popen` object:

```python
p.communicate()
```

this will wait until the process terminates avoid deadlocks if the `stderr` buffer is too large.

Reply:
Good point.  It should be a simple call but we could get weird results without that call to `communicate`.  Thanks for the observation.  I'm adding that right now and making sure it continues to work like I expect since I'm no Python expert.